### ISSUE TITLE
Plague Attribute

### ISSUE THREAD

**Issue by jay1148** [Cardshifter/Cardshifter#264]
Wanted to throw another attribute idea out there. Plague attribute would act like death touch from MTG, only instead of killing the unit that attacked the unit that possesses the plague attribute, the player who initiated the attack against the unit that possesses the plague attribute would take damage directly equal to the attack of the unit that possesses the plague attribute (or equal to the unit that attacked).

Example:
Bear vs Snake (with P attribute)

Bear attacks snake, owner of bear would take damage equal to bear's attack power.
or
Bear attacks snake, owner of bear would take damage equal to snake's attack power.

What do you all think, and which one would work better?

**Comment by Zomis** [comment]
Should Plagued attribute only have effect when you're *defending*? If the Snake attacks Bear, then nothing should happen, right?

**Comment by jay1148** [comment]
Yes, it is only when attacked that the attribute effect activates.

**Comment by Zomis** [comment]
Related to #266
